drivers/sensor/nxp/nxp_tempsense/Kconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config NXP_TEMPSENSE
+	bool "NXP temperature sensor (TEMPSENSE)"
+	default y
+	depends on DT_HAS_NXP_TEMPSENSE_ENABLED
+	select FPU if CPU_HAS_FPU
```

Comment:
seems this is not necessary? there is not floating value calculation in this driver.